src/main/java/org/eln2/mc/Eln2.kt

```diff
@@ -36,4 +41,5 @@ object Eln2 {
             ModStatistics.sendAnalytics()
         }
     }
+
```

Comment:
?

Reply:
IntelliJ added this when it formatted the code on commit